wps-cloudformation-template.yaml

```diff
@@ -60,6 +66,15 @@ Parameters:
   workerRetryIntervalMs:
     Type: String
     Default: '60000'
+  geonetworkCatalogueURL:
+    Type: String
+    Default: 'https://catalogue-imos.aodn.org.au/geonetwork'
+  geonetworkLayerSearchField:
+    Type: String
+    Default: 'layer'
+  provenanceTemplateS3Key:
```

Comment:
We can get rid of this parameter and can workout the value from ConfigurationFileMap and s3configKey.